Sources/StringExtractor/ExtractionError.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public enum ExtractionError: Error {
-    public struct Context {
+  public struct Context: Codable, Sendable {
```

Comment:
This probably should've just been `Sendable`... must've accidentally typed `Codable` first 🙈